nhs/utils/logging.py

```diff
@@ -36,19 +37,46 @@ def emit(self, record: logging.LogRecord) -> None:
         )
 
 
-def config_logger() -> None:
+def read_config(config_path: str = "configurations.yml") -> Dict[str, Any]:
     """
-    Configure loguru logger settings and set it as as the default logger
+    Read YAML configuration file and return as a dictionary.
+
+    Args:
+        config_path (str): Path to the configuration file. Defaults to 'configurations.yml'.
+
+    Returns:
+        Dict[str, Any]: Configuration settings as a dictionary.
+
+    Raises:
+        FileNotFoundError: If the configuration file is not found.
+        yaml.YAMLError: If there's an error parsing the YAML file.
```

Comment:
I'm assuming co-pilot generated this - AI is fine but check that the format is consistent with the code style. See wiki on writing documentation https://github.com/zazwaz12/CITS3200---National-Housing-Simulation/wiki/Contributing-to-the-Project#coding-guidelines

Reply:
Yeah ur right, done